### ISSUE TITLE
OpenTrace, BlueTrace, TraceTogether dostępny jako kod źródłowy

### ISSUE THREAD

**Issue by kierepka** [ProteGO-app/specs#97]
https://github.com/opentrace-community 
Dobrze przetestowana i opisana aplikacja publicznie dostępna - może jednak z nich wziąć przykład?

**Comment by Saibamen** [comment]
Po 2-10 commitów w repozytoriach... Na dyskietkach pracowali przed publikacją? :D Przecież tu żadnej historii nie ma, a przeglądając zmiany w kodzie można się bardzo  dużo nauczyć o projekcie

**Comment by broukema** [comment]
Searching through the links at https://en.wikipedia.org/wiki/COVID-19_apps might help to find more of the history of OpenTrace which at en.Wikipedia redirects to https://en.wikipedia.org/wiki/BlueTrace . My speculation is that the Singapore developers convinced their government to free-licence and grab a good "open" sounding name before others could. Or maybe it was even free-licensed right from the start. Anyway: https://en.wikipedia.org/wiki/TraceTogether is apparently GPLv3 (not just kod źródłowy, but free-licensed source code).

This thread overlaps a bit with https://github.com/ProteGO-app/specs/issues/73 where I added a brief analysis (as a non-expert, just reading/editing the page) of the tables listing different protocols and software packages. There are several different free-licensed protocols and software packages worth comparing.

**Comment by qLb** [comment]
Yes, Opentrace, BlueTrace and even TraceTogether (not so safe at all i suspect) have very interesting design principles and use cases. At the first stage (before G+A tracing code release/review is publicly available) we activly develop the technical spec and tests of few features included in the publicly available resources of some of those technologies. Because in the future releases we'd like to swap all libraries used for tracing to G+A compatible and surely it's going to be a breaking change we're not focusing on delivering those features right now, in this instant, to the public - but rather test them for their capabilities for future implementation fo aforementioned designs.

**Comment by potiuk** [comment]
https://techcrunch.com/2020/04/24/apple-and-google-update-joint-coronavirus-tracing-tech-to-improve-user-privacy-and-developer-flexibility/
